feedback-module/src/components/Module.js

```diff
@@ -74,16 +74,24 @@ function Module({ pagetitle, endpoint, dir }) {
   }, [screen]);
 
   // requestInfo parses the user's info and returns the object that will be submitted to the Microsoft Flow endpoint
-  const requestInfo = (formID) => {
-    if (ENDPOINTS.includes(formID)) {
-      trackFormAction(formID);
-      setCheckedOptions(checkedFields);
-      setInputResponses(inputQuestions);
+  const requestInfo = (formID, feedbackID, text) => {
+    let submission;
+    setFeedbackType(en(text), feedbackID);
```

Comment:
Do we need to call all these functions (setFeedbackType, trackFormAction, setCheckedOptions, setInputResponses) even if the formID is "interaction"?

Reply:
The functions all have a check for whether or not to actually set the new values (e.g. `setCheckedOptions` checks whether or not there are any checkboxes on the page), so they don't actually do anything on the first screen since the checkboxes and input fields on that page are undefined.

Reply:
But if they aren't doing anything then why would we even call them?

Reply:
They're being called on the other screens when there are checkboxes and input fields— just thought it'd be more flexible for future use if we didn't make the calls dependent on any specific `formID` values.

Reply:
Yeah I guess so. It just seems like we already have so many things that are dependent on the type on formID values so this wouldn't be that different. Because the track interactions call is very specific so it never really would be a part of a screen that has checkboxes or input boxes